### ISSUE TITLE
[A11y] Backend Dashboard Skip to Content link not visible

### ISSUE THREAD

**Issue by wallyOSU** [OregonDigital/OD2#3286]
### Operating System

Windows

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Skip to Content link not visible when using keyboard to navigate. [WCAG guidelines](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G1) state Skip to Content link needs to either be visible at all times, or visible upon keyboard focus.

[Recorded example](https://media.oregonstate.edu/media/t/1_fy7nb1sy) of Skip to Content being read by NVDA, but not visible or highlighted.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Use keyboard to navigate page. Ensure that Skip to Content link is visible when focused.
